core/src/marker/markerManager.cpp

```diff
@@ -72,6 +72,11 @@ bool MarkerManager::setBitmap(MarkerID markerID, int width, int height, const un
     texture->setData(bitmapData, size);
 
     marker->setTexture(std::move(texture));
+
+    // The geometry is unchanged, but the mesh must be rebuilt because DynamicQuadMesh contains
+    // texture batches as part of its data.
+    buildGeometry(*marker, m_zoom);
```

Comment:
I wrote that at first, but I think it could actually be kind of misleading. For instance, if you call `setBitmap` on a marker before you set the styling or geometry, then the call would return `false` even though the bitmap was successfully assigned to the marker.